src/parameterized/LKJL.jl

```diff
@@ -24,8 +24,20 @@ struct LKJL{k, N, T} <: ParameterizedMeasure{N}
     par :: NamedTuple{N,T}
 end
 
+LKJL{k}(nt::NamedTuple{N,T}) where {k,N,T} = LKJL{k,N,T}(nt)
+
 LKJL(k,η) = LKJL{k, (:η,), Tuple{typeof(η)}}((η=η,))
```

Comment:
With the below constructor, this can be simpler
```suggestion
LKJL(k,η) = LKJL{k}(η)
```